File: OOPS/Encapsulation/encapsulation.cpp
#include <bits/stdc++.h>
using namespace std;
class Student{
    public:
   string getname(){
        return name;
    }
    void setname(string x){
     name = x;   
    }
    int getroll(){
        return roll;
    }
    void setroll(int x){
         roll = x;
    }
    private:
      int roll;
      string name;
};

int main() {
        Student s1 =  Student();
	    	s1.setroll(2);
	    	s1.setname("isha");
		
	    	cout<<(s1.getroll());
	    	cout<<(s1.getname());
        return 0;
}

File: OOPS/Encapsulation/encapsulation.java
class Student{
    private int roll;
    private String name;
    public String getname(){
        return name;
    }
    public void setname(String name){
     this.name = name;   
    }
    public int getroll(){
        return roll;
    }
    public void setroll(int roll){
         this.roll = roll;
    }
}
class Main{
    public static void main(String[] args) {
        Student s1 = new Student();
		    s1.setroll(2);
		    s1.setname("Isha");
		
		    System.out.println(s1.getroll());
		    System.out.println(s1.getname());
    }
}

//Output: 2 Isha 
